examples/Actor/DemoActor/build-container.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Given the commands within the scripts are the same across platforms, I'd just add the commands directly to the `README` to avoid having to maintain them.

Reply:
Done